content/blog/rails-dynamic-translation-constants.md

```diff
@@ -0,0 +1,156 @@
++++
+title = "Dynamic Translations in Rails: Why Constants Can’t Adapt to Locale Changes and How to Fix It?"
+slug = "rails-dynamic-translation-constants"
+date = 2024-11-04T10:35:08+05:30
+image = "/images/2023/ruby-generic.png"
+draft = true
+authors = ["Nitin Rajkumar"]
+description = ""
+tags = ["Playbook", "Software Craftsmanship", "Rails", "I18n", "Translations Not Reflecting", "Numeronyms"]
```

Comment:
You can remove the "Translations not reflecting" and "Numeronym" tags
Also will be a good idea to keep the tags and categories the same